sample_rock/rockcraft.yaml

```diff
@@ -0,0 +1,37 @@
+name: flask-app
+summary: A flask sample app
+description: OCI image for the sample flask app
+version: "0.1"
+base: bare
+build-base: ubuntu:22.04
+license: Apache-2.0
+platforms:
+  amd64:
+
+parts:
+  flask-environment:
+    plugin: nil
+    overlay-script: |
+      mkdir $CRAFT_OVERLAY/etc
+      mkdir $CRAFT_OVERLAY/tmp
+      chmod 755 $CRAFT_OVERLAY/etc
+      chmod 777 $CRAFT_OVERLAY/tmp
+      groupadd -R $CRAFT_OVERLAY --gid 2000 flask
+      useradd -R $CRAFT_OVERLAY --system --gid 2000 --uid 2000 --home /srv/flask flask
+  flask-dependencies:
+    stage-packages:
+      - python3-dev
+    build-packages:
+      - build-essential
+      - python3-pip
+    plugin: nil
```

Comment:
I'm wondering if the `python` plugin would work here

Reply:
I tried to use the python plugin, but the python plugin does not work properly without a `setup.py`.